### PR TITLE
Added cheat sheet and playground site from our coffee shop day

### PR DESCRIPTION
**Pull request by amiller-gh**

### CODE REVIEW

**Comment by amiller-gh** [comment]
@melissaroman, could use a review 😃

File: cheatsheet.js [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+console.log('asdf' == true);   // Log: true
+console.log('asdf' === true);  // Log: false
+console.log('' == false);      // Log: true
+console.log('' === false);     // Log: false
+console.log(1 == true);        // Log: true
+console.log(1 === true);       // Log: false
+console.log(0 == false);       // Log: true
+console.log(0 === false);      // Log: false
+
+// Fun Fact:
```
Comment (amiller-gh) [inline cheatsheet.js:147]: I don't know if this is the right place for this explanation... but I don't know where else to put it!!

File: cheatsheet.js [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+// All logic expressions will evaluate to the boolean values `true` or `false`
+// depending on the conditions to the left and right side.
+
+// The `==` operator checks for *EQUALITY* and evaluates to `true` or `false`
+console.log(true == true);     // Log: true
+console.log(true == false);    // Log: false
+console.log(10 == 20);         // Log: false
+console.log('asdf' == 'asdf'); // Log: true
+
+// The `===` operator checks for *type safe* equality and evals to `true` or `false`.
```
Comment (amiller-gh) [inline cheatsheet.js:134]: Not sure if this is explained super well...

File: cheatsheet.js [hunk 3]
```diff
@@ -0,0 +1,300 @@
...
+    var c = 3;
+    console.log(a, b, c) // Logs: 4, 2, 3
+  }
+
+  two();
+  console.log(a, b) // Logs: 1, 2, undefined
+}
+
+console.log(a, b, c); // Logs 1, undefined, undefined
+one();
```
Comment (amiller-gh) [inline cheatsheet.js:300]: TODO: Add control blocks here! (Like `if`, `else if`, `else`, `while`, `for`, etc...)

File: README.md [hunk 4]
```diff
@@ -1 +1,12 @@
-# Hi Trev!
\ No newline at end of file
+# Hi Trev!
+## Welcome to your development playground repository!
+
+You should probably read up about Markdown, that is what any file with a `.md` file extension contains – like this one!
+
+[Read about markdown here!](https://guides.github.com/features/mastering-markdown/)
```
Comment (melissaroman) [inline README.md:6]: And maybe just bookmark this page - nothing you need to memorize but nice to have a reference!

File: cheatsheet.js (hunk 2 above)
Comment (melissaroman) [reply cheatsheet.js:134]: I think it explains it well. `===` is like it is truly equal in all aspects. I could be wrong about this but from my experience, it seems like I almost always use `===`.

File: cheatsheet.js (hunk 1 above)
Comment (melissaroman) [reply cheatsheet.js:147]: This works here! And is a fun fact!! 😄

File: cheatsheet.js [hunk 5]
```diff
@@ -0,0 +1,300 @@
...
+// The `<` operator checks if the left value is *Less Than* the right value.
+console.log(10 < 5);  // Log: false
+console.log(5 < 10);  // Log: true
+console.log(10 < 10); // Log: false
+
+// The `>` operator checks if the left value is *Greater Than* the right value.
+console.log(10 > 5);  // Log: true
+console.log(5 > 10);  // Log: false
+console.log(10 > 10); // Log: false
+
```
Comment (melissaroman) [inline cheatsheet.js:176]: Yay math!!!! 🎉

File: cheatsheet.js [hunk 6]
```diff
@@ -0,0 +1,300 @@
...
+// - Subtraction Operator
+console.log(3 - 2) // Log 1
+
+// * Multiplication Operator
+console.log(2 * 2) // Log 4
+
+// \ Division Operator
+console.log(2 / 2) // Log 1
+
+
```
Comment (melissaroman) [inline cheatsheet.js:204]: @amiller-gh Add in modulo here?
Modulo (written using %) is basically the remainder:

30 / 3 = 10, remainder 0
=> `30 % 3 = 0`

10 / 3 = 3, remainder 1
=> `10 % 3 = 1`

File: cheatsheet.js [hunk 7]
```diff
@@ -0,0 +1,300 @@
...
+ *******************************************************/
+
+// Functions take a block of code and save it for later.
+// The code inside the curly brackets will not execute until you "call" this function!
+// You can declare a function like this:
+function myFunctionName() {
+  console.log('Called this function!');
+}
+
+// The above *technically* is shorthad for this (you typically won't do this):
```
Comment (melissaroman) [inline cheatsheet.js:216]: nitpick...but spelling... `shorthand`

File: cheatsheet.js [hunk 8]
```diff
@@ -0,0 +1,300 @@
...
+  var b = 2;
+
+  function two() {
+    var a = 4;
+    var c = 3;
+    console.log(a, b, c) // Logs: 4, 2, 3
+  }
+
+  two();
+  console.log(a, b) // Logs: 1, 2, undefined
```
Comment (melissaroman) [inline cheatsheet.js:296]: I think you meant to have `c` in the console.log()

File: index.html [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+    <link rel="stylesheet" type="text/css" href="./style.css">
+  </head>
+
+  <!-- The BODY element contains everything you actually see in the browser. -->
+  <body>
+
+    <!-- HTML consists of nested "elements" that may have zero to many "attributes" -->
+    <!-- Every element has a different purpose. These H1 and H2 elements below are headers. Headers are numbered H1 - H6. -->
+    <h1>Trevor's Awesome Website!</h1>
+    <h2>Itsn't it cool?</h2>
```
Comment (melissaroman) [inline index.html:26]: Spelling... `Isn't` 😉

File: script.js [hunk 10]
```diff
@@ -0,0 +1,90 @@
...
+    name: "Lion's Roar",
+    type: "broadsword",
+    strength: 8,
+  },
+  yell() {
+    console.log('I FIGHT FOR MY FRIENDS!!');
+  }
+};
+
+function takeDamage(ammount, target) {
```
Comment (melissaroman) [inline script.js:42]: It doesn't matter since `ammount` is used throughout but just because it's me and spelling: `amount`. 😄

File: style.css [hunk 11]
```diff
@@ -0,0 +1,30 @@
+/*
+   This is a CSS comment!
+   They are ignored by the browser.
+   I'll leave you notes in here using these!
+
+   TODO: Annotate a CSS demo...
+*/
```
Comment (melissaroman) [inline style.css:7]: Just a note to @lemonhawk21 that @amiller-gh knows like EVERY CSS property...but that's not normal. He's a CSS (and general coding) wizard so keep that in mind when you need to look things up that he somehow just knows.

**Comment by melissaroman** [comment]
Other than a few minor things I commented on, this looks great!!! And is a _really_ good intro/overview of the basics!

File: cheatsheet.js (hunk 6 above)
Comment (amiller-gh) [reply cheatsheet.js:204]: Ooh, good call!

File: cheatsheet.js (hunk 7 above)
Comment (amiller-gh) [reply cheatsheet.js:216]: 😛

File: index.html (hunk 9 above)
Comment (amiller-gh) [reply index.html:26]: ITS INTENTIONAL! 😝

Comment (melissaroman) [reply index.html:26]: Is it? 😏

File: style.css (hunk 11 above)
Comment (amiller-gh) [reply style.css:7]: download

**Pull request merged**